text/0000-Project-Goals-2025h2.md

```diff
@@ -0,0 +1,754 @@
+| :--                                                                                      | :--           | :--                 | :--      | :-- |
+| [C++/Rust Interop Problem Space Mapping](https://rust-lang.github.io/rust-project-goals/2025h2/interop-problem-map.html)                         |               |                     |          |     |
+| [Const Generics](https://rust-lang.github.io/rust-project-goals/2025h2/const-generics.html)                                                      |               | \*3                 |          |     |
+| [Continue Experimentation with Pin Ergonomics](https://rust-lang.github.io/rust-project-goals/2025h2/pin-ergonomics.html)                        |               | ✅                   |          |     |
+| [Design a language feature to solve Field Projections](https://rust-lang.github.io/rust-project-goals/2025h2/field-projections.html)             | \*5           | \*4                 | ✅        |     |
+| [Develop the capabilities to keep the FLS up to date](https://rust-lang.github.io/rust-project-goals/2025h2/FLS-up-to-date-capabilities.html)    |               |                     |          |     |
+| [Ergonomic ref-counting: RFC decision and preview](https://rust-lang.github.io/rust-project-goals/2025h2/ergonomic-rc.html)                      |               | \*7                 | \*8      |     |
+| [Evolving trait hierarchies](https://rust-lang.github.io/rust-project-goals/2025h2/evolving-traits.html)                                         |               |                     |          | \*6 |
+| [Finish the std::offload module](https://rust-lang.github.io/rust-project-goals/2025h2/finishing-gpu-offload.html)                               | ![Complete][] |                     |          |     |
+| [Getting Rust for Linux into stable Rust: language features](https://rust-lang.github.io/rust-project-goals/2025h2/Rust-for-Linux-language.html) |               |                     |          |     |
+| ↳ Finish and stabilize `arbitrary_self_types` and `derive_coerce_pointee`                |               |                     |          | ✅   |
+| [In-place initialization](https://rust-lang.github.io/rust-project-goals/2025h2/in-place-initialization.html)                                    |               | Two design meetings |          |     |
+| [MIR move elimination](https://rust-lang.github.io/rust-project-goals/2025h2/mir-move-elimination.html)                                          |               |                     | ✅        |     |
+| [Nightly support for Autoreborrow traits](https://rust-lang.github.io/rust-project-goals/2025h2/autoreborrow-traits.html)                        |               |                     |          |     |
+| ↳ Design autoreborrow internals                                                          | \*2           |                     | ✅        |     |
```

Comment:
I think this one should focus on the experiment, and perhaps design meeting to surface open questions and get feedback. This is a design that I think would benefit a lot from an experiment to prove it out ahead of accepting an RFC, so it's unlikely we can do so this year. cc @aapoalas

Reply:
Put in suggested changes to the three lines to go along these lines more.

Reply:
The suggested changes were added in https://github.com/rust-lang/rust-project-goals/pull/380